concrete/src/Routing/RouteGroupBuilder.php

```diff
@@ -175,7 +175,9 @@ protected function processMiddlewares(Route $route)
 
     protected function processScope(Route $route)
     {
-        $route->setOption('oauth_scopes', $this->scope);
+        if (!$route->getOption('oauth_scopes')) {
+            $route->setOption('oauth_scopes', $this->scope);
```

Comment:
I wonder if this is what we want. I'd guess we want these scopes to merge instead of choosing one set or the other

Reply:
Yeah, right now they're overwritten entirely; I think we can continue with this and add merging in the future.